src/Corax/IndexWriter.cs

```diff
@@ -1851,6 +1890,38 @@ void ThrowTriedToDeleteTermThatDoesNotExists()
             entriesForTerm.Dispose(_entriesAllocator);
 
             _indexMetadata.Increment(indexedField.NameTotalLengthOfTerms, totalLengthOfTerm);
+
+            long TryGetNullTermPostingList(CompactTree tree, out long termContainerId, out CompactTree.CompactKeyLookup keyLookup, out bool termExists, ref EntriesModifications entries, out bool alreadyIndexed)
+            {
+                keyLookup = default;
+                termContainerId = ~long.MaxValue | (tree.RootPage << 3); // sign bit means its null, rootpage identificates field
```

Comment:
That may be a problem, since we are using varints here, no?
Which means that this would take a _lot_ of values.
Why do we need this to be anything but `0` ?